SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/DormancyAndTombstoneTest/DormancyTestActor.cpp

```diff
@@ -8,13 +8,8 @@
 
 ADormancyTestActor::ADormancyTestActor()
 {
-	bReplicates = true;
 	TestIntProp = 0;
 
-	GetStaticMeshComponent()->SetStaticMesh(LoadObject<UStaticMesh>(nullptr, TEXT("StaticMesh'/Engine/BasicShapes/Sphere.Sphere'")));
```

Comment:
Why remove this?

Reply:
Random drive-by. We now have the ReplicatedTestBase actor, which has a bunch of these useful things, such as a mesh and some basic replication, so we can remove some code duplication.